cli/tools/repl/session.rs

```diff
@@ -44,6 +44,33 @@ Object.defineProperty(globalThis, "_error", {
 });
 
 globalThis.clear = console.clear.bind(console);
+
+globalThis.repl = {
+  help: () => {
+    console.log("%cAvailable functions:", "font-style: italic");
+    console.log("%crepl.help()", "color: green;", "                 Print this message");
+    console.log("%crepl.reload()", "color: green;", "               Create a new session without exiting the REPL");
+    console.log("%crepl.exit()", "color: green;", "                 Exit the REPL");
+    console.log("%crepl.save(maybeFilename)", "color: green;", "    Save the current session to a file");
+    console.log("%cclear()", "color: green;", "                     Clear screen");
```

Comment:
Maybe we should document `close()` here instead of `repl.exit()`, but still have a `repl.exit()`? (maybe should be named `repl.close()` to match `close()`)

Reply:
Done